src/components/StateCityDropdowns.tsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect } from 'react';
+import { UseFormRegister, UseFormWatch } from 'react-hook-form';
+
+import { useCitiesLazyQuery } from '../generated/graphql';
+import { IFormInputs } from '../pages/CarForm';
+import Dropdown from './Dropdown';
+
+interface DropdownProps {
+  states:
+    | {
+        id: number;
+        name: string;
+      }[]
+    | undefined;
+  isErrorState: boolean;
+  isErrorCity: boolean;
+  register: UseFormRegister<IFormInputs>;
+  watch: UseFormWatch<IFormInputs>;
+}
+
+const BrandModelDropdowns = ({
+  states,
+  isErrorState,
+  isErrorCity,
+  register,
+  watch,
```

Comment:
same comment about nested fields in hook form, I shared the docs about this above

Reply:
I solved it